src/pages/home/report/ReportActionItem.js

```diff
@@ -244,6 +246,7 @@ class ReportActionItem extends Component {
                         toggleReaction={this.toggleReaction}
                     />
                 )}
+                <ReportActionItemThread numberOfReplies={7} mostRecentReply={'1:03pm'} icons={ReportUtils.getIcons(this.props.report, this.props.personalDetails)} />
```

Comment:
@chiragsalian how are we handling the conditional rendering of this. Would we just check if `this.props.action.childReportID` exists?

Reply:
ReportUtils.js isThread?

Reply:
Discussed 1:1, yes just checking `this.props.action.childReportID` should suffice to render this item.